tests/v2/validation/provisioning/airgap/k3s_custom_cluster_test.go

```diff
@@ -81,7 +80,9 @@ func (a *AirGapK3SCustomClusterTestSuite) SetupSuite() {
 }
 
 func (a *AirGapK3SCustomClusterTestSuite) TestProvisioningK3SCustomCluster() {
-	a.clustersConfig.NodesAndRoles = []machinepools.NodeRoles{provisioninginput.AllRolesPool}
+	a.clustersConfig.MachinePools = []provisioninginput.MachinePools{
+		provisioninginput.AllRolesMachinePool,
```

Comment:
nit; why not just leave this one like you already had?

Reply:
oh, you mean the formatting right? I can revert that.